src/run.php

```diff
@@ -62,6 +62,7 @@
     } else {
         $log->info('There are no input tables.', ['runId' => $runId]);
     }
+    $fs->chmod($validator->getDataDir(), 0777, 0000, true);
```

Comment:
for readability, can you put this right after the mkdirs ?

Reply:
Not really, because it proceeds to make files and scripts and I need those to all have permissions

Reply:
@odinuv I added a comment there that maybe helps.

Reply:
hmm, so if the rstudio/jupyter creates a file, will the data-loader be able to read it ?

Reply:
no, but it doesn't need to, it only runs on start up.  The API can read the files though

Reply:
right! data-loader doesn't have to, but data-loader-api has to